lib/cylc/gui/updater.py

```diff
@@ -195,7 +195,16 @@ def reconnect(self):
         try:
             self.daemon_version = self.suite_info_client.get_info(
                 'get_cylc_version')
-        except (ConnectionError) as exc:
+        except ConnectionDeniedError as exc:
+            if cylc.flags.debug:
+                traceback.print_exc()
+            if not self.connect_fail_warned:
+                self.connect_fail_warned = True
+                gobject.idle_add(
+                    self.warn,
+                    "ERROR: %s\n\nIncorrect suite passphrase?" % exc)
+            return
+        except ConnectionError as exc:
```

Comment:
I have moved the `ConnectionDeniedError` before the `ConnectionError` block here, because `ConnectionDeniedError` is an instance of `ConnectionError`.

The `ConnectionDeniedError` block would never catch anything before this change.